services/pub/pub-publisher.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  publisherId: Joi.string().allow(null).required(),
+}).required()
+
+class PubPublisher extends BaseJsonService {
+  static category = 'other'
+
+  static route = {
+    base: 'pub/publisher',
+    pattern: ':packageName',
+  }
+
+  static examples = [
+    {
+      title: 'Pub Publisher',
+      namedParams: { packageName: 'dart.dev' },
+      staticPreview: {
+        label: 'publisher',
+        message: 'dart.dev',
+        color: 'blue',
+      },
+      keywords: ['dart', 'dartlang'],
+    },
+  ]
+
+  static defaultBadgeData = { label: 'publisher' }
+
+  async fetch({ packageName }) {
+    return this._requestJson({
+      schema,
+      url: `https://pub.dev/api/packages/${packageName}/publisher`,
+    })
+  }
+
+  async handle({ packageName }) {
+    const data = await this.fetch({ packageName })
+    const publisher = data.publisherId
+    return {
+      label: 'publisher',
+      message: publisher == null ? 'unverified' : publisher,
+      color: publisher == null ? 'red' : 'blue',
+    }
+  }
+}
+
+export { PubPublisher }
```

Comment:
Just a heads up since I imagine this was mirrored off the module for the other pub badge (and admittedly a nit), but we typically do default class exports unless we have something like a redirector defined in the same module and need multiple exports

Reply:
Hmm, so I don't need this explicit export line?

Reply:
Moved the `export` to be in front of the PubPublisher declaration.